RecoMuon/GlobalTrackingTools/interface/DynamicTruncation.h

```diff
@@ -107,6 +107,7 @@ class DynamicTruncation {
   void                 useSegment(CSCSegment const &, TrajectoryStateOnSurface const &);
   void                 sort(ConstRecHitContainer&);
   void                 setEtaRegion();
+  static edm::ParameterSetDescription fillDefaultThrsParameters();
```

Comment:
it looks like some commits were missed.
I do not see an implementation of this method.
Please remove to avoid undefined symbols.

Reply:
I forgot to remove this line